### PR TITLE
feat: enable filtering on dates for general feature cards

### PR DESCRIPTION
**Pull request by tcheee**

### CODE REVIEW

File: src/hooks/useStrapi.ts [hunk 1]
```diff
@@ -147,6 +147,15 @@ export const useStrapi = <T>({
       'uid',
     );
     apiUrl.searchParams.set('filters[PersonalizedFeatureCard][$nei]', 'true');
+    //filter url
+    const currentDate = new Date(Date.now()).toISOString().split('T')[0];
```
Comment (oktapodia) [inline src/hooks/useStrapi.ts:151]: I'd love to use a date formatter instead of this litte hack, but not blocking

File: src/hooks/useStrapi.ts [hunk 2]
```diff
@@ -147,6 +147,15 @@ export const useStrapi = <T>({
...
+    apiUrl.searchParams.set(
```
Comment (oktapodia) [inline src/hooks/useStrapi.ts:152]: We need to swap to the strapi api soon-ish to make it cleaner and resilient imo

**Review by oktapodia — APPROVED** [review]
LGTM

**Review by dennyscode — CHANGES_REQUESTED** [review]
Once I set startCampaign (older than date of today) on strapi that Feature-Card is not shown.

**Review by dennyscode — APPROVED** [review]

**Pull request merged**